src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/support/DeadlineSupportController.java

```diff
@@ -0,0 +1,175 @@
+package uk.gov.hmcts.cmc.claimstore.controllers.support;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.hmcts.cmc.claimstore.exceptions.NotFoundException;
+import uk.gov.hmcts.cmc.claimstore.services.ClaimService;
+import uk.gov.hmcts.cmc.claimstore.services.DirectionsQuestionnaireService;
+import uk.gov.hmcts.cmc.claimstore.services.UserService;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponse;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ClaimantResponseType;
+import uk.gov.hmcts.cmc.domain.models.claimantresponse.ResponseRejection;
+import uk.gov.hmcts.cmc.domain.models.response.Response;
+import uk.gov.hmcts.cmc.domain.models.response.ResponseType;
+import uk.gov.hmcts.cmc.domain.models.response.YesNoOption;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+@RestController
+@RequestMapping("/support/deadline")
+public class DeadlineSupportController {
+    private static final String CLAIM_DOES_NOT_EXIST = "Claim %s does not exist";
+
+    private static final Predicate<Claim> CREATED_BEFORE_5_0_0 = claim -> claim.getCreatedAt()
+        .isBefore(LocalDateTime.of(2019, Month.SEPTEMBER, 9, 15, 12));
```

Comment:
I've recently had to use this date to calculate things too. It may be an idea to put this in a utils class so it can be defined in one place

Reply:
Moved to `DateUtils`